source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/CircleViewModel.cs

```diff
@@ -468,6 +476,25 @@ private void CreateCircle()
                     Point2 = null; 
                     HasPoint2 = false;
                     ResetFeedback();
+
+                    // Set map extent to extent of circle if manually entered
+                    if (!interactiveMode)
```

Comment:
like you did in Pro, move this redundant code to a method that takes IGeometry and any other needed params